save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/testsuite/TestSuiteType.kt

```diff
@@ -1,25 +0,0 @@
-package com.saveourtool.save.testsuite
-
-import kotlinx.serialization.Serializable
-
-/**
- * Types of test suites
- */
-@Serializable
-enum class TestSuiteType {
-    /**
-     * Type Obsolete Standard for old test suites
-     */
-    OBSOLETE_STANDARD,
```

Comment:
What will happen with the periodic task for test suites update?

Reply:
they will create a new version, and we will take that latest available test suites source for standard run.
clean-up of old version will be done in phase 3 (or 4 :smile:)